HOW_TO_WRITE_FEATURE_DETECTS.md

```diff
@@ -36,20 +53,114 @@ The following code represents an example of the schema (it does not represent a
+|------------------|:---------:|:----------------------------------------------------:|:---------------------------------------------------:|
+| `name`           |  required |             Name of the feature detection            |                                                     |
+| `property`       |  required | The property name established in `Modernizr.addTest` |    It must be lowercase, without any punctuation    |
+| `tags`           |  optional |    A group that encapsulates many feature detects    |                                                     |
+| `caniuse`        |  optional |      A conversion table of caniuse and Modernizr     | Consider adding [caniuse testing](#caniuse-testing) |
+| `authors`        |  optional |          List of contributors of the script          |                                                     |
+| `builderAliases` |  optional |     Used by CI and the web when tests are renamed    |          Should not be needed in new tests          |
+| `polyfills`      |  optional |     Available polyfills for not working versions     |      Check the [polyfills section](#polyfills)      |
+| `aliases`        |  optional |   Used if a feature has more than a canonical name   |               Legacy only - do not use              |
+| `async`          |  optional |       If the test supports async functionality       |                  Defaults to false                  |
+| `warnings`       |  optional |        Notes to the developer using the script       |            Don't mistake it for knownBugs           |
+| `knownBugs`      |  optional |  Bugs known of the test (e.g.: doesn't work in IE6)  |                                                     |
+| `notes`          |  optional |                  Links to resources                  |                                                     |
+
+## Polyfills
+
+Polyfills can be included in the [metadata](#metadata) with a simple word. In order to do this the [lib/polyfills.json](lib/polyfills.json) must include the description of all polyfills.
+
+### Polyfill Schema
+
+The following example represents the schema in which polyfills must be defined in the [polyfills.json](lib/polyfills.json) file:
+
+```json
+"css3pie": {
+  "name": "CSS3 PIE",
+  "authors": ["Jason Johnston"],
+  "href": "http://css3pie.com/",
+  "licenses": ["Apache2", "GPL2"],
+  "notes": ["CSS3 decoration rendering for IE 6-9. Supports: border-radius, box-shadow, multiple backgrounds, linear gradients, border-image"]
+},
+```
+
+> Metadata does no need to appear in a set order, however, it is common to follow the order stated above.
+
+### Polyfill Item Description
+
+|            | Necessity |                       Description                      |
+|------------|:---------:|:------------------------------------------------------:|
+| `name`     |  required | Name of the polyfill used in the [metadata](#metadata) |
```

Comment:
Okay, this is completely wrong

Reply:
in what way?

Reply:
The name that you put in the name attribute is the one that appears on the web (if I'm not wrong), the name that it is used to name polyfills in the metadata is the head of the branch.
In this example you name the polyfill with `css3pie` not with `CSS3 PIE`, the latter one would be the prettified way that appears on the web.

Reply:
Solved in 6f838e7